regression/contracts/trivial_contract_replace/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=10$
+^SIGNAL=0$
+^\[main.assertion.\d+\] line \d+ assertion foo\(\&n\) != 15\: FAILURE$
+^VERIFICATION FAILED$
+--
+--
+Check whether CBMC doesn't crash when replacing incomplete contracts, i.e.,
+without requires using ensures and vice-versa.
```

Comment:
Similar comment as above. In the test, we are not checking the other direction: "requires without ensures" but we say "vice-versa", so do you want to add another function for checking this direction as well?

Reply:
Implemented as suggested.